src/main/scala/org/ergoplatform/ErgoBox.scala

```diff
@@ -46,36 +44,40 @@ class ErgoBox private(
                        override val additionalTokens: Seq[(TokenId, Long)] = Seq(),
```

Comment:
@kushti I think it is time to revise scaladoc for ErgoBox and sync it with the implementation.